webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -59,7 +59,19 @@ import { selectRepositorySetsStatus } from './RepositorySetsSelectors';
 import './RepositorySetsTab.scss';
 import SortableColumnHeaders from '../../../../Table/components/SortableColumnHeaders';
 import SelectableDropdown from '../../../../SelectableDropdown';
+import { hasRequiredPermissions, hostIsNotRegistered, missingRequiredPermissions, userPermissionsFromHostDetails } from '../../hostDetailsHelpers';
 
+const requiredViewPermissions = [
+  'view_hosts', 'view_activation_keys', 'view_products',
```

Comment:
If you don't have all 3 of these permissions, the Repository sets API won't even _show_ you anything. So I just hide the tab.

When the tab moves to Content this will need some refactoring..